transformers/struct.go

```diff
@@ -228,9 +225,10 @@ func (t *structTransformer) ignoreField(field reflect.StructField) bool {
 	switch {
 	case len(field.Name) == 0,
 		slices.Contains(t.skipFields, field.Name),
-		!field.IsExported(),
 		isTypeIgnored(field.Type):
 		return true
+	case !field.IsExported():
```

Comment:
this is required as the embedded structs might contain other embedded structs and so on